tequilapi/contract/node.go

```diff
@@ -84,6 +84,42 @@ type ProviderConsumersCountResponse struct {
 	Count int `json:"count"`
 }
 
+// ProviderSeriesEarningsResponse reflects a earnings data series metrics during a period of time.
+// swagger:model ProviderSeriesEarningsResponse
+type ProviderSeriesEarningsResponse struct {
+	Data []ProviderSeriesEarningItem `json:"data"`
+}
+
+// ProviderSeriesEarningItem reflects a MYSTT earnings and timestamp
+type ProviderSeriesEarningItem struct {
+	Earning   float64 `json:"m"`
+	Timestamp int64   `json:"t"`
```

Comment:
maybe it would bake sense to have: x and y or c (count) and t (timestamp) or something commonly generic because it's now (m, t), (c, t) or (b, t). "m" doesn't even correspond to first letter of Earning. Also this would be unifying.

Reply:
@mdomasevicius Yes, it make sense to use common values for these chart endpoints.
I can use (c, t) for all metrics. But we can move forward and for minimizing json size, I can use just array [[c, t],...[c, t]]. So what way preferred for you?

Reply:
It's local requests, no need to minimizing JSON size, make it readable and convenient.

Reply:
I see, so will be use {"c", "t"} objects

Reply:
I would go with a `value` and `timestamp`, it's not really `count`.

Reply:
Yes, it make sense.